### DISCUSSION TITLE
PNG image loader

### DISCUSSION THREAD

**Discussion by mohammad-torabi-84** [moononournation/Arduino_GFX#331] [Q&A] [answered]
Hi,

I tried to compile the PNG image loader in `Arduino_GFX/examples/ImgViewer/ImgViewerPng/ImgViewerPng.ino`. but it could not compile. there is an error in `PNGDraw()` function. it calls draw16bitRGBBitmap() with 6 arguments while in Arduino_RBG_Display class this function expected 5.

any suggestion?

**Comment by moononournation** [comment]
it should be similar to #328 , will be fixed.

**Reply by mohammad-torabi-84** [reply]
So, is the solution on that thread is valid?

**Reply by moononournation** [reply]
The workaround is valid, and soon will be fixed.

**Comment by moononournation** [comment, marked as answer]
Please get update and try again.
